pkg/rest/client.go

```diff
@@ -155,6 +160,10 @@ func (d *dynatraceClientImpl) ReadById(api Api, id string) (json []byte, err err
 
 	response, err := get(d.client, url, d.token)
 
+	if !success(response) {
+		return nil, fmt.Errorf("Failed to get existing configs for api %v (HTTP %v)!\n    Response was: %v", api.GetId(), response.StatusCode, string(response.Body))
```

Comment:
```suggestion
		return nil, fmt.Errorf("Failed to get existing config for api %v (HTTP %v)!\n    Response was: %v", api.GetId(), response.StatusCode, string(response.Body))
```

I'd assume `ReadById` gets a single object and this line is copy paste from somewhere else that read several